hyperspy/io_plugins/hdf5.py

```diff
@@ -425,9 +425,9 @@ def parse_structure(key, group, value, _type, compression):
             try:
                 group.attrs[key] = value
             except:
-                print("The hdf5 writer could not write the following "
-                      "information in the file")
-                print('%s : %s' % (key, value))
+                warnings.warn(
```

Comment:
Looking at the python docs, I guess we should here use `_logger.execption()` since we're suppressing an exception?